### ISSUE TITLE
Can only boot after cleannvram

### ISSUE THREAD

**Issue by hellviz** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#39]
Hi and thanks for this great repo! Everything worked great from start but after a few days I suddenly couldn’t boot. The bar stuck in the beginning and rebooted the computer. Tried to check in verbose mode to se where it got stuck but it looked random. Tried different settings in config and stumbled over the cleannvram tool. After using that the system booted without a problem. After that I tried different boot args to see if it could boot within cleannvram. After some tries it started working but it still seemed random. It worked for a few days and then the same error occurred and I had to use the cleannvram tool again. Do you have any tips? I have the same specs as you but a simpler graphics card.

**Comment by SchmockLord** [comment]
Check your BIOS-settings again: https://github.com/SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D/blob/master/bios-settings.md

And download a fresh copy of my config. Just add "agdpmod=pikera" to the bootargs and replace the serial numbers with yours. You shouldn't do any more than that and it should work if you have the same board.

**Comment by hellviz** [comment]
Hi Chris and thanks for the reply!

I’ve double checked the BIOS settings and tried to add the agdpmod arg but still no success. It must be something else. What exactly does the cleannvram tool do? What I’ve understood you shouldn’t need that anymore and OC would handle it internally but for some reason doing a cleannvram is the only way I can boot.

/Henrik

Skickat från min iPhone

28 juli 2020 kl. 22:47 skrev Chris S. <notifications@github.com>:

﻿

Check your BIOS-settings again: https://github.com/SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D/blob/master/bios-settings.md

And download a fresh copy of my config. Just add "agdpmod=pikera" to the bootargs and replace the serial numbers with yours. You shouldn't do any more than that and it should work if you have the same board.

**Comment by SchmockLord** [comment]
This is hard to tell from a distance. Could be 1000 things. Maybe try a clean install again?

We can also do a personal session via Skype/teamviewer. See my Patreon page: https://www.patreon.com/SchmockLord

**Comment by hellviz** [comment]
Yeah I know. I have tested some different boot-args but I guess I have to try to do a clean install.
Thanks for the help!

/Henrik

**Comment by hellviz** [comment]
Hi!

I managed to get it to boot with my vanilla config. Tried ti compare the configs ang changed som values but still no luck with your config. Instead I copied your aml files and that did the trick. Don’t know exactly what fixed it but hey, it works:)

I attached my config if you’re curious.

Thanks again!

/Henrik



29 juli 2020 kl. 22:30 skrev Chris S. <notifications@github.com<mailto:notifications@github.com>>:



This is hard to tell from a distance. Could be 1000 things. Maybe try a clean install again?

We can also do a personal session via Skype/teamviewer. See my Patreon page: https://www.patreon.com/SchmockLord
